app/app-controllers/error-screen/errorController.js

```diff
@@ -73,7 +73,7 @@ define([
         this.viewPlugin.off('back');
     };
 
-    ErrorController.prototype._generateErrorCallback = function(errorType, params) {
```

Comment:
Removing `errroType` here is breaking line [99](https://github.com/mobify/astro-scaffold/pull/125/files#diff-9e8c7e3e8735d4c25296f68a27665b19L99) below.